docs/source/contribute.rst

```diff
@@ -17,5 +17,16 @@ from source checkout.
 
 We like high test coverage and mypy_ type annotations.
 
+Making releases
+---------------
```

Comment:
What do you think about adding a note that contributors shouldn't do this themselves, that this is a memo for the maintainers? It is inconvenient to receive pull requests which bump package version, in addition to implementing a feature.

Reply:
Good point. What do you think of the following right after the header:

```
Note: releases are made from master by eli5 maintainers.
When contributing a pull request, please do not update release notes
or package version.
```

Reply:
:+1: